src/helpers/helpers.js

```diff
@@ -950,6 +950,26 @@ module.exports.register = function (Handlebars, options, params) {
         },
 
         /**
+        * Consistently generate the edit link for a file in VSTS
+        **/
+        editInVSTSLink: function (options) {
+            var src = options.hash.src || (typeof this.page !== 'undefined' ? this.page.src : '');
+            return [
+                stache.config.vsts_protocol,
+                stache.config.vsts_base,
+                '/',
+                stache.config.vsts_project,
+                '/_git/',
+                stache.config.vsts_repo,
+                '?path=%2F',
```

Comment:
That `%2F` is an encoded forward slash.  Looking at what VSTS expects, I would remove the leading `%2F` and instead send `src` through the `encodeURIComponent` function.